.github/workflows/chronos-notebook-python-spark.yml

```diff
@@ -0,0 +1,60 @@
+name: Chronos Notebook Python Spark py37
+
+# Cancel previous runs in the PR when you push new commits
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.run_id }}
+  cancel-in-progress: true
+
+# Controls when the action will run. 
+on:
+  schedule:
+    - cron: '30 15 * * *' # GMT time, 15:30 GMT == 23:30 China
+  # Trigger the workflow on pull request events but only for the main branch
+  pull_request:
+    branches: [ main ]
+    paths:
+      - '.github/workflows/chronos-notebook-python-spark.yml'
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  Chronos-example:
+    runs-on: [ self-hosted, Gondolin, ubuntu-20.04-lts ]
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.7.10"]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 8
+        uses: ./.github/actions/jdk-setup-action
+      - name: Set up Maven
+        uses: ./.github/actions/maven-setup-action
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Run Chronos use-case notebook
+        shell: bash
+        run: |
+          conda remove -n chronos-prvn-env -y --all
+          conda create -n chronos-prvn-env -y python==3.7.10 setuptools==58.0.4 -c ${GONDOLIN_CONDA_CHANNEL} --override-channels
+          source activate chronos-prvn-env
```

Comment:
we should test `source bigdl-nano-init` in later NB tests (no need to revise in this PR)